lib/api/collection_config.rb

```diff
@@ -73,6 +73,13 @@ def name_for_klass(resource_klass)
       @cfg.detect { |_, spec| spec[:klass] == resource_klass.name }.try(:first)
     end
 
+    def name_for_subklass(resource_klass)
```

Comment:
I don't like `subklass` bit.  you can just say `subclass` here and it won't anything (as well as `resource_class` for that matter).  `klass` is only necessary when standalone because it's a keyword.